src/pages/OwnerPage/Header/HeaderBanners/ExceededUploadsAlert/ExceededUploadsAlert.jsx

```diff
@@ -1,19 +1,21 @@
+import PropTypes from 'prop-types'
+
 import A from 'ui/A'
 import Banner from 'ui/Banner'
 import BannerContent from 'ui/Banner/BannerContent'
 import BannerHeading from 'ui/Banner/BannerHeading'
 import Button from 'ui/Button'
 
-const ExceededUploadsAlert = () => (
+const ExceededUploadsAlert = ({ planName, monthlyUploadLimit }) => (
```

Comment:
How do you feel about using hooks in this component that get the information you need rather than providing those as parameters? We've tried to adopt this notion that components can and should be* standalone, so they can used anywhere you need them without it's parent needing a specific interface, leading to prop-drilling. 

My first instinct is then to be "well, I already have the information on my parent, what's the big deal, plus it it's another request anyway". To the 1st point, this leads to coupling and prop drilling, which we try to avoid. To the 2nd point, this approach doesn't lead to any extra requests/latency on our end because useQuery caches requests with a particular queryKey, so it won't fire N requests for N calls from a component and all of it's children, it would instead fire once and on any subsequent requests check if there's a cached response from that. Wdyt?

If you agree, then my suggestion would be to add the `usePlanData` hook and get the planName/monthlyUploadLimit from there (and adjust the tests and whatnot). This applies to the other component below as well.

Reply:
That makes sense to me. Thanks for the explanation.